runtime/interpreter/statictype.go

```diff
@@ -659,6 +660,29 @@ func (Unauthorized) Equal(auth Authorization) bool {
 	return ok
 }
 
+type Inaccessible struct{}
+
+var InaccessibleAccess Authorization = Inaccessible{}
+
+func (Inaccessible) isAuthorization() {}
+
+func (Inaccessible) String() string {
+	return ""
```

Comment:
Would it be possible to return a description here? At the same time, could `Unauthorized` above also get a non-empty string representation?